.github/workflows/build.yml

```
@@ -23,10 +23,14 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-${{ matrix.ruby }}-gem-
 
+      - name: Install bundler
+        run: gem install bundler:2.1.4
+
       - name: Bundle Install
-        run: |
-          bundle config path vendor/bundle
-          bundle install --jobs 4 --retry 3
+        run: bundle check --path vendor/bundle || bundle install --path vendor/bundle --jobs 4 --retry 3
```

```suggestion
        run: BUNDLE_PATH=vendor/bundle bundle check --path vendor/bundle || bundle install --jobs 4 --retry 3
```

This avoids a warning.